packages/clarity-js/src/core/measure.ts

```diff
@@ -1,12 +1,13 @@
 import { Setting } from "@clarity-types/core";
 import { Metric } from "@clarity-types/data";
+import { report } from "@src/core/report";
 import * as metric from "@src/data/metric";
 
 // tslint:disable-next-line: ban-types
 export default function (method: Function): Function {
     return function (): void {
         let start = performance.now();
-        method.apply(this, arguments);
+        try { method.apply(this, arguments); } catch (ex) { throw report(ex); }
```

Comment:
Should we apply this in other places where we think the errors would be of interest?

Reply:
Thanks for pointing it out. I have intentionally kept this scoped for now - to not change existing behavior. Majority of code flows through this path for performance measurement. Once we have ensured this scoped change hasn't caused any regression - we can then expand to more code paths and cover core.start and data.start methods.